mhs/README.md

```diff
@@ -60,10 +60,21 @@ When running the tests locally, you will need to set the MHS_ADDRESS and ASID in
     - eg MHS_ADDRESS=localhost will be resolved as 'http://localhost/'
 
 ## Running an MHS Instance
-`pipenv run mhs` will run the `main.py` script (you can also run this directly). This will start up an MHS
-instance listening for 'client' requests on port 80 and asynchronous responses from Spine on port 443.
-Note that the following environment variables need to be set when running MHS:
-- `MHS_LOG_LEVEL` - log level threshold
+Since the MHS is split into multiple services the simplest way to run an instance locally is to use the docker-compose script
+found in the root directory. The following steps should be followed: 
+* Requirements: `Docker`, [`Packer`](https://www.packer.io/)
+* Run the `./build.sh` script found in the top level directory of this project. This will build the inbound and outbound  
+    containers ready for running
+* Run `docker-compose build`. This build all the additional containers
+* Run `docker-compose up`. This will start the inbound and outbound services, along with an instance of RabbitMQ and 
+    DynamoDb
+* Depending on the OS the MHS is being run on, measures may have to be taken to allow public inbound traffic so that
+the async responses from Spine can access the MHS. For example on windows a inbound rule was required in windows
+firewall to allow inbound traffic through port 443. In the AWS test environment an inbound rule was added
+in the security groups for machine the MHS was deployed to
```

Comment:
Think you have to enable the inbound rules for that specific instance even if there is a vpn connection, otherwise spine is blocked sending the async response. There is a rule on the build box to allow this traffic through